packages/compass-import-export/src/modules/export.spec.js

```diff
@@ -50,24 +45,65 @@ describe('export [module]', () => {
         });
       });
 
-      afterEach(function(done) {
-        rimraf(tempFile, done);
+      describe('CSV Export', () => {
+        let tempFile;
+        beforeEach(() => {
+          tempFile = path.join(
+            os.tmpdir(),
+            `test-${Date.now()}.csv`
+          );
+        });
+        afterEach((afterEachDone) => {
+          rimraf(tempFile, afterEachDone);
+        });
+        it('should set the correct fields to CSV export', (done) => {
```

Comment:
suggestion: we are trying to standardize towards `async/await` for async code, I know this is a test, so is not that big of a deal, but would it be possible to `promisify` or use Promise based api here instead of `done`?